### DISCUSSION TITLE
Use of official SystemRDL logo

### DISCUSSION THREAD

**Discussion by jeras** [SystemRDL/systemrdl-compiler#199] [Ideas]
[Accellera freely grants and encourages](https://www.accellera.org/about/policies-and-procedures/logo-use) the use of secondary logos used for branding individual standards in projects implementing those standards. So you could/should add the logo to the front page.

![SystemRDL logo](https://www.accellera.org/images/about/policies/logos/SystemRDL-logo.png)

**Comment by jeras** [comment]
As a follow up, is there a SystemRDL compiler logo I could use in a presentation?

**Reply by amykyta3** [reply]
I have hesitated to use the SystemRDL logo mostly because the digital quality of it is quite poor and would need to be cleaned up. Even in this post, it doesn't hold up well when displayed in a dark browser theme:
![image](https://github.com/SystemRDL/systemrdl-compiler/assets/12021539/f670f674-4baa-4c6b-a19d-a8f31b43d616)
The SystemRDL 1.0 spec includes source files for it, but they are all of similarly low resolution, over-compressed, and generally poor quality. It is also notably absent from the 2.0 spec, so I had assumed they kinda abandoned the logo since it does look a tad dated too.


For the SystemRDL compiler project, I dont really have an official "logo" other than the Github avitar emblem. You are free to use that to signify this project space (but not for derivative projects).
The vector file can be found here: https://github.com/SystemRDL/.github/blob/main/profile/logo.svg

**Reply by jeras** [reply]
Thanks.

**Reply by chrismcdev** [reply]
@jeras @amykyta3 feel free to use this one here: ![SystemRDL Logo](https://raw.githubusercontent.com/chrismcdev/PeakRDL-check/main/.github/assets/systemrdl-logo.svg)
